requirements.txt

```diff
@@ -42,6 +46,7 @@ pydantic==1.8.2
 pymongo==3.12.3
 pyparsing==3.0.4
 pytest==6.2.4
+pytest-asyncio==0.18.1
```

Comment:
What about adding dev-dependencies like `pytest-asyncio` in [requirement-dev](https://github.com/fast-recsys/movie-recommender-backend/blob/main/requirements_dev.txt)?

Reply:
Makes sense. Moved this to dev-dependencies. But I don't know which other packages we'd need only during dev to be honest since dependencies of dependencies are also listed in the `requirements.txt` file. Is there any easy way to figure this out?

Reply:
Hm I see. It's possible, and I am sharing a solution we used in my company:

We created two requirements file, [dev.txt and prod.txt which was keeping main packages only](https://github.com/AOT-Technologies/forms-flow-ai/tree/master/forms-flow-api/requirements).

Then requirments.txt contained what was the requirement when installing with a [MakeFile like this](https://github.com/AOT-Technologies/forms-flow-ai/blob/master/forms-flow-api/Makefile#L42-L43)